deploy/crds/k8s.nginx.org_nginxingresscontrollers_crd.yaml

```diff
@@ -181,6 +181,11 @@ spec:
               required:
               - enable
               type: object
+            enableLatencyMetrics:
```

Comment:
I believe this file is autogenerated. So it should have the description  Enable collection of latency metrics for upstreams" taken from pkg/apis/k8s/v1alpha1/nginxingresscontroller_types.go